projects/igniteui-angular/src/lib/tabs/tabs.component.ts

```diff
@@ -358,6 +381,14 @@ export class IgxTabsComponent implements IgxTabsBase, AfterViewInit, OnDestroy {
         }
     }
 
+    private setGroupsAttributes() {
+        for (let index = 0; index < this.groups.length; index++) {
+            const tabsGroup = Array.from(this.groups)[index] as IgxTabsGroupComponent;
```

Comment:
Create a const with the groups array before the for to avoid creating the array on each iteration.